desc/objectives/objective_funs.py

```diff
@@ -980,6 +1012,54 @@ def jac_unscaled(self, *args, **kwargs):
         """Compute Jacobian matrix of vector form of the objective wrt x, unweighted."""
         return self._jac_unscaled(*args, **kwargs)
 
+    def jvp_scaled(self, v, x, constants=None):
+        """Compute Jacobian-vector product of the objective function.
+
+        Uses the scaled form of the objective.
+
+        Parameters
+        ----------
+        v : tuple of ndarray
+            Vectors to right-multiply the Jacobian by.
+        x : ndarray
+            Optimization variables.
+        constants : list
+            Constant parameters passed to sub-objectives.
+
+        """
```

Comment:
probably a stupid question but there are jvp_scaled methods for _Objective now and the one already in ObjectiveFunction, is the idea that if you want a single objective's derivative you'd call the _Objective method, but the ObjectiveFunction is still separate and the jvp computed using its own compute_scaled method?

Reply:
yeah exactly. The `_Objective` jvp methods get used in the proximal thing when the `ObjectiveFunction` deriv mode is blocked.